package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@undone-labs/fontpls",
```

Comment:
`npm i -g undone-labs/fontpls` is a little lengthy (what's in the readme is ideal)

Reply:
@orvn I can unpublish and republish without the org name but that would publish it not under the undone-labs organization on npm. The format `@<org-name>/<package-name>` for the name in the `package.json` seems to be the thing that actually scopes the npm package under the org

Reply:
@orvn ping ^

Reply:
Note: in the ticket comments it was decided that the package would be unpublished from npm under the `undone-labs` organization and republished by @orvn